src/lib.rs

```diff
@@ -73,6 +73,9 @@ pub enum Error {
 /// // Relative paths are not supported
 /// assert_eq!(windows_to_wsl("Program Files (x86)\\Foo\\bar.txt").unwrap_err(), Error::RelativePath);
 /// assert_eq!(windows_to_wsl("..\\foo\\bar.txt").unwrap_err(), Error::RelativePath);
+///
+/// // Windows WSL paths are converted to the root
+/// assert_eq!(windows_to_wsl("\\\\?\\UNC\\wsl.localhost\\distro\\home\\user\\file").unwrap(), "/home/user/file");
```

Comment:
Can you also add a test for the error case (i.e. host name is not `wsl.localhost`)?

Reply:
Ok, fixed. 

I'm not sure if generic network paths could be supported in the future, they probably need to be mounted somewhere, so it might not be possible. But for now, they remain unsupported.